src/workloads/execution/FilterWithLogicalExpressions.yml

```diff
@@ -0,0 +1,162 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
```

Comment:
Can you add keywords as well?

Reply:
I am not sure I added the correct keywords, but I tried to follow how other workloads did it.